Library/Formula/flink.rb

```diff
@@ -0,0 +1,18 @@
+class Flink < Formula
+  desc "Platform for scalable batch and stream data processing"
+  homepage "https://flink.apache.org"
+  head "https://github.com/apache/flink.git"
+  url "https://www.apache.org/dyn/closer.cgi?path=flink/flink-0.9.0/flink-0.9.0-bin-hadoop2.tgz"
```

Comment:
It's probably fine given it's a Java thing. Presumably does need a `:java` dependency though?